erpnext/selling/doctype/sales_order_item/sales_order_item.json

```diff
@@ -10,6 +10,7 @@
   "item_code",
   "customer_item_code",
   "ensure_delivery_based_on_produced_serial_no",
+  "is_stock_item",
```

Comment:
Is this required? It looks like you're checking the cached value from item during validation?